### ISSUE TITLE
A11y_SQLAzureDataStudio_ServerDashboard_ ServerDashboard _Screenreader: Wrong focus order is observed on Sever Dashboard screen while Voiveover is on

### ISSUE THREAD

**Issue by shweta255** [microsoft/azuredatastudio#9288]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOS;#MAS1.3.2;#Screenreader;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3
Additional Details:
Mac OS:10.15.3
MAS References: MAS1.3.2

**Repro Steps:**
1.	Launch Azure Data Studio.
2.	Connect to server.In the Connections pane, right-click on a server & select Manage.
3.	Launch Voiceover
4.	Use Control +Option +right arrow key and try to navigate to toggle more button.

**Actual:**
While Voiveover is on and navigated to ‘Toggle More’ button of Sever Dashboard screen using Control+ option and arrow key, wrong focus order. For e.g. If navigating to ‘Search’ section, Screenreader focus moves to ‘Toggle more’ button first and then to ‘Search’ plain text.

Same is the case when tried to navigate to Reverse direction. Wrong focus order is observed.

**Same issue is observed in Database Dashboard screen:** 1. Launch Azure Data Studio.2. Connect to server.3. In the Connections pane, navigate to server --> database --> right click on any database & select Manage.

**Expected:**
While Voiveover is on and navigated to ‘Toggle More’ button of Sever Dashboard screen using Control+ option and arrow key, screenreader focus should be moved to that particular section’s text first and then to ‘Toggle more’ button. For e.g. If navigating to ‘Search’ section, Screenreader focus should be moved to ‘Search’ plain text first and then to ‘Toggle more’.

**User Impact:**
Screenreader users will face difficulty as they will not be able to understand for which context ‘Toggle more’ button is present.

**Attachment link for Reference:** 
[9288_A11y_SQLAzureDataStudio_ServerDashboard_ ServerDashboard _Screenreader.zip](https://github.com/microsoft/azuredatastudio/files/4274848/9288_A11y_SQLAzureDataStudio_ServerDashboard_.ServerDashboard._Screenreader.zip)

**Comment by alanrenmsft** [comment]
we need to adjust the order of the elements defined in the html template to match the visual order, we used float: right here that caused this.

**Comment by sspatil123** [comment]
GithubTags:#Closed; Version 1.41.1
![image](https://user-images.githubusercontent.com/85863999/216962445-143c4fbd-14fc-42b8-ad9c-8a0334f94395.png)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
